listenbrainz_spark/utils.py

```diff
@@ -264,3 +264,12 @@ def read_json(hdfs_path, schema):
     """
     df = listenbrainz_spark.session.read.json(config.HDFS_CLUSTER_URI + hdfs_path, schema=schema)
     return df
+
+def upload(hdfs_path, local_path):
```

Comment:
since this is in utils, `upload_to_hdfs` would be a much clearer name. Earlier, it was called as `hdfs_connection.client.upload` which gave enough context, but `utils.upload` doesn't really help in understanding where it's getting uploaded.

Reply:
thanks, will make the change :)